app/src/main/java/com/conupods/OutdoorMaps/View/SearchView/SearchActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.conupods.OutdoorMaps.View.SearchView;
+
+import androidx.fragment.app.FragmentActivity;
+import androidx.recyclerview.widget.DefaultItemAnimator;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.widget.LinearLayout;
+import android.widget.SearchView;
+import android.widget.Toast;
+
+import com.conupods.OutdoorMaps.Services.ActivityComponentBuilder;
+import com.conupods.OutdoorMaps.Models.Buildings.AbstractCampusLocation;
+import com.conupods.OutdoorMaps.Services.CampusAbstractLocationCreationService;
+import com.conupods.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchActivity extends FragmentActivity implements CampusLocationsAdapterListener{
+
+    private final String TAG = "SeacrhcActivity";
+    private List<AbstractCampusLocation> mCampusLocationList;
+    private RecyclerView recyclerView;
```

Comment:
mRecyclerView

Reply:
was made into a local variable instead since its never used outside the onCreate() method